docs/Microservices/RPS/securityRPS.md

```diff
@@ -17,7 +18,7 @@ In addition to the above assets, there are best practices that are recommended t
 
 ### 1 Remote Admin Password
```

Comment:
I'm confused by these numbers that start headings. Am I missing something?

Reply:
removed numbers